tm2/pkg/store/rootmulti/proof_test.go

```diff
@@ -116,37 +116,6 @@ func TestVerifyMultiStoreQueryProof(t *testing.T) {
 	require.NotNil(t, err)
 }
 
-func TestVerifyMultiStoreQueryProofEmptyStore(t *testing.T) {
```

Comment:
ics23 compatibility: this test is commented because proofs on empty store are no longer supported.
See related SDK comment here https://github.com/cosmos/cosmos-sdk/pull/6390/files#diff-3c078909c57ced350f0aaf3c18cfab5f13a184a6a940345189543ead9d8132e4L112